From 00900688f3e1ad670c53eaf69c1c8b5e3a4ec08e Mon Sep 17 00:00:00 2001
From: Patrick Spendrin <ps_ml@gmx.de>
Date: Mon, 2 Dec 2013 00:37:46 +0100
Subject: [PATCH 23/56] The F12 shortcut is reserved on Windows

Since we do not want to hack windows, we simply change the default to F11.
---
 app/mainwindow.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/app/mainwindow.cpp b/app/mainwindow.cpp
index 9b0a987..ca5df80 100644
--- a/app/mainwindow.cpp
+++ b/app/mainwindow.cpp
@@ -192,7 +192,11 @@ void MainWindow::setupActions()
     action = actionCollection()->addAction("toggle-window-state");
     action->setText(i18nc("@action", "Open/Retract Yakuake"));
     action->setIcon(KIcon("yakuake"));
+#ifndef Q_OS_WIN
     action->setGlobalShortcut(KShortcut(Qt::Key_F12));
+#else
+    action->setGlobalShortcut(KShortcut(Qt::Key_F11));
+#endif
     connect(action, SIGNAL(triggered()), this, SLOT(toggleWindowState()));
 
     action = actionCollection()->addAction("keep-open");
-- 
1.9.3

